ember-model.js

```diff
@@ -340,29 +340,29 @@ Ember.ManyArray = Ember.RecordArray.extend({
   },
 
   replaceContent: function(index, removed, added) {
-    added = Ember.EnumerableUtils.map(added, function(record) {
+    added = added.map(function(record) {
       return record._reference;
     }, this);
 
     this._super(index, removed, added);
   },
 
-  _contentWillChange: function() {
+  _contentDidChange: function() {
```

Comment:
Is there is an open PR on ember-model for this or that they have already been merged? If merged, any reason why we didn't branch off master?